tasks/main.yml

```
@@ -34,6 +34,28 @@
   import_tasks: ca.yml
   when: ca_ca_host == inventory_hostname
 
+- name: Set temporary variable ca_san to hostname, FQDN, and inventory hostname
+  set_fact:
+    ca_san: "DNS:{{ ansible_hostname }},DNS:{{ ansible_fqdn }},DNS:{{ inventory_hostname }}"
+
+- name: Add IP addresses and loopback address for etcd >= 3.2.0 to ca_san
+  set_fact:
+    ca_san: >-
+      {{ ca_san }},
+      IP:127.0.0.1,
+      {%- for host in vars['play_hosts'] -%}
+        IP:{{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}
+        {%- if not loop.last -%}
+          ,
+        {%- endif -%}
+      {%- endfor -%}
+  when: ca_etcd | bool
+
+- name: Combine ca_san and ca_san_custom if defined
+  set_fact:
+    ca_san: "{{ca_san + ca_san_custom }}"
```

```suggestion
    ca_san: "{{ ca_san + "," + ca_san_custom }}"
```
Linter wants more spaces. 🙂 
By adding a , you spare the user to start their string with a comma. I just hope, I got the syntax right.